test/suites/ecma6-tests/ecma6-constants-and-variables-tests.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2013 - present UTN-LIS
```

Comment:
Review indentation. Please take note of unsupported cases for Puma. This will help to define future tasks on the runtime support.